pkgs/json_syntax_generator/test_data/pubspec_lock/pubspec_lock.schema.json

```diff
@@ -82,11 +82,13 @@
             "properties": {
                 "name": {
                     "type": "string",
+                    "pattern": "^[a-zA-Z_]\\w*$",
```

Comment:
uber-nit: for consistency with the rest move this after the description key?

Reply:
Ah, let me add the jsons in this package to pkgs/hooks/tool/normalize.dart then they will also be consistent everywhere! (We should probably move that normalizer somewhere else.)